diff --git weblate/formats/convert.py weblate/formats/convert.py
index f3df4d9..896a627 100644
--- weblate/formats/convert.py
+++ weblate/formats/convert.py
@@ -307,7 +307,7 @@ class MarkdownFormat(ConvertFormat):
     def save_content(self, handle):
         """Store content to file."""
         converter = MarkdownTranslator(
-            inputstore=self.store, includefuzzy=True, outputthreshold=None, maxlength=80
+            inputstore=self.store, includefuzzy=True, outputthreshold=None, maxlength=100000000000000000
         )
         templatename = self.template_store.storefile
         if hasattr(templatename, "name"):
